lib/src/models/transaction_uploader.dart

```diff
@@ -131,18 +132,23 @@ class TransactionUploader {
     //  throw StateError('Unable to validate chunk: $_chunkIndex');
 
     // Catch network errors and turn them into objects with status -1 and an error message.
-    final res = await _api.post('chunk', body: json.encode(chunk));
-
-    _lastRequestTimeEnd = DateTime.now().millisecondsSinceEpoch;
-    lastResponseStatus = res.statusCode;
-
-    if (lastResponseStatus == 200) {
-      _chunkIndex++;
-    } else {
-      lastResponseError = getResponseError(res);
-      if (FATAL_CHUNK_UPLOAD_ERRORS.contains(lastResponseError)) {
-        throw StateError(
-            'Fatal error uploading chunk: $_chunkIndex: $lastResponseError');
+    Response? res;
+    try {
+      res = await _api.post('chunk', body: json.encode(chunk));
+    } catch (e) {
+      print("Error posting to /chunk endpoint:, " + e.toString());
+    }
+    if (res != null) {
+      _lastRequestTimeEnd = DateTime.now().millisecondsSinceEpoch;
```

Comment:
This value impacts the amount of delay we apply between failed posts... Should it be hoisted outside this `if`?

Reply:
[Thanks for catching that](https://github.com/ardriveapp/arweave-dart/pull/14/commits/ce5d2453300dd47077711a84979803939c8175dc)